ui/main/src/app/business/view/useractionlogs/userActionLogs.view.ts

```diff
@@ -97,12 +97,12 @@ export class UserActionLogsView {
         this.pageNumber = pageNumber;
     }
 
-    public search(): Observable<UserActionLogsResult> {
+    public search(isForExport: boolean): Observable<UserActionLogsResult> {
```

Comment:
set default value to false , it will avoid to set the value in test for example

Reply:
Done.